### DISCUSSION TITLE
Fix the overflowing app icons in the channel header with the Apps Bar

### DISCUSSION THREAD

**Discussion by esethna** [mattermost/community#19] [Practical Ideas]
This is getting crazy!

![image](https://user-images.githubusercontent.com/14333569/122953748-b9dd1a00-d333-11eb-9523-a40878271445.png)

The **Apps Bar** organizes all your apps neatly and easily accessible and allows users to choose what they want to see there with pinning, like google chrome extensions: 

![image](https://user-images.githubusercontent.com/14333569/122953462-987c2e00-d333-11eb-9b62-d87105f08aa6.png)

**Comment by andrewbrown00** [comment]
Agreed @esethna, from a UX perspective it's really hard to follow and know what's new, why it was added, are the new icons temporary, etc. All reasons to support adding a mechanism, the Apps Bar, to allow users to have more control/customization and consistency for apps.
